packages/anon-aadhaar-pcd/src/utils.ts

```diff
@@ -265,3 +266,27 @@ export async function exportCallDataGroth16FromPCD(
   }
   return { a, b, c, Input }
 }
+
+/**
+ * Fetch the public key PEM file from the serverless function endpoint.
+ * @param url Endpoint URL to fetch the public key.
+ * @returns {Promise<string | null>} The official Aadhaar public key.
+ */
+export const fetchPublicKey = async (
```

Comment:
I see we have 2 fetchPublicKey function, another one in packages/anon-aadhaar-pcd/src/prover.ts